content/files/rate.js

```diff
@@ -85,8 +85,15 @@ function do_loaddata() {
       var issues = [];
       for (j = 3; j < information.length; j += 2) {issues.push({id: information[j], count: information[j + 1]})}
       issues.sort(function (a, b) {return b.count - a.count;});
-      for (j = 0; j < issues.length; j++)
-        txt += issues[j].count + '&times;<a href="https://issues.jenkins.io/browse/JENKINS-' + issues[j].id + '">JENKINS-' + issues[j].id + '</a> ';
+      for (j = 0; j < issues.length; j++) {
+        if (!isNaN(parseInt(issues[j].id))) {
+          txt += `${issues[j].count}&times;<a href="/issue/${issues[j].id}">JENKINS-${issues[j].id}</a> `;
```

Comment:
change to use the issue redirect, won't work in local dev run setup but will work in production and gets us ready for the auto redirect to GitHub if we know about the issue

(relates to https://github.com/jenkins-infra/jenkins.io/pull/8572)